wdl/PlotSVCountsPerSample.wdl

```diff
@@ -174,10 +178,13 @@ task CatOutliersPreview {
             outliers_dict[sample].add(reason)
           else:
             outliers_dict[sample] = {reason}
-    with open("~{prefix}.outliers_preview.samples.txt", 'w') as OUT:
-      OUT.write("#sample\treason\n")
+    with open("~{prefix}.outliers_preview_with_reason.samples.tsv", 'w') as with_reason, \
+        open("~{prefix}.outliers_preview.samples.txt", 'w') as OUT, open("num_outliers.txt", 'w') as num:
+      with_reason.write("#sample\treason\n")
       for sample in sorted(list(outliers_dict.keys())):
-        OUT.write(sample + "\t" + ",".join(outliers_dict[sample]) + "\n")
+        with_reason.write(sample + "\t" + ",".join(outliers_dict[sample]) + "\n")
+        OUT.write(sample + "\n")
+      num.write(str(len(outliers_dict)))
```

Comment:
I probably would have put the `with` statement for `num` right here for readability, but it's fine

Reply:
Thanks, I will leave as-is for now so that I can merge this, but I will keep this in mind for the future